components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Controller/DmApiPublisherMethodControllerTests.cs

```diff
@@ -24,12 +24,32 @@ namespace Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Tests.Engine {
     using Serializers.NewtonSoft;
     using Xunit;
     using static Microsoft.Azure.IIoT.Modules.OpcUa.Publisher.Agent.PublisherJobsConfiguration;
+    using Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Storage;
+    using System;
 
     /// <summary>
     /// Tests the Direct Methods API for the pubisher
     /// </summary>
     public class DmApiPublisherControllerTests {
 
+        private readonly string _tempPublishedNodesFile;
+
+        public DmApiPublisherControllerTests() {
```

Comment:
Create a base class that will take care of providing temp file to inherited classes.

Reply:
Added.